src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -36,8 +36,9 @@ public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
             validateName(trustDataDto.getTrustName(), errors, loggingContext);
             validateCreationDate(trustDataDto.getCreationDate(), errors, loggingContext);
 
-            if (isFullValidation) {
-                validateCeasedDate(overseasEntitySubmissionDto, trustDataDto, errors, loggingContext);
+            if (isFullValidation &&
```

Comment:
the ceased date can also be present if the 'is trust still involved' question is missing, as the web page can show the ceased date without the yes/no question if there are no BO's with NOC trustee of a trust.

Reply:
Not sure what should be done in response to this here.

Reply:
In this case surely the web will just set this flag to false and send it to the api. I'm not sure this can be dealt with in the api as it only sees the json sent, it cannot see the web page and won't know the difference between this and when the flag has just not been set.